onnx/defs/nn/defs.cc

```diff
@@ -2871,25 +2896,40 @@ ONNX_OPERATOR_SET_SCHEMA(
               "Rotate using interleaved pattern. Default value is 0 (False).",
               AttributeProto::INT,
               OPTIONAL_VALUE)
+        .Attr("rotary_embedding_dim",
+              "Rotary embedding dimension used to apply partial rotary embeddings. Default value is 0. ",
```

Comment:
How is a default value of 0 useful or meaningful? Perhaps preferable to not have a default value?